requirements/pytorch/test.txt

```diff
@@ -1,6 +1,6 @@
 coverage>=6.4, <=6.4.2
 codecov>=2.1, <=2.1.12
-pytest>=7.0, <=7.1.2
+pytest>=7.0, <7.1.4
```

Comment:
@akihironitta shouldn't the test requirements have the min version match the max version?

Reply:
I think you're right. With our current CI setup, having the lower bounds in the test requirements (e.g. `pytest>=7.0` in this case) has no meaning. (we want our CI to always use the latest test packages even in "oldest" settings.)

We could make it:
```
pytest<7.1.4  # or pytest<=7.1.3
```
or make it even stricter like this:
```
pytest==7.1.3
```

In any way, it should have the same version installed in our current setup. What do you think? (I'm fine with any of these)

Reply:
If `==` works well with dependabot, that would be the clearest option

Reply:
```suggestion
pytest ==7.1.3
```